content/en/user-guide/aws/kms/index.md

```diff
@@ -106,6 +106,24 @@ $ awslocal kms create-key --tags '[{"TagKey":"_custom_id_","TagValue":"00000000-
 }
 {{< / command >}}
 
+## Custom Key Material for KMS Keys via Tags
+
+You can seed a KMS key with custom key material using the `_custom_key_material_` tag during creation. This can be useful to pre-seed a development environment so values encrypted with localstack KMS can be decrypted later.
```

Comment:
Create two separate lines for this: 

```
You can seed a KMS key with custom key material using the `_custom_key_material_` tag during creation. 
This can be useful to pre-seed a development environment so values encrypted with localstack KMS can be decrypted later.
```